app/components/Secrets/Generic/Generic.jsx

```diff
@@ -462,22 +512,22 @@ class GenericSecretBackend extends React.Component {
                                         floatingLabelText="Filter"
                                         hintText="Filter list items"
                                         onChange={(e, v) => {
-                                            this.setState({
-                                                filteredSecretList: _.filter(this.state.secretList, (item) => {
-                                                    return item.includes(v);
-                                                })
-                                            })
+                                            let filtered = v ? _.filter(this.state.fullSecretList, (item) => { return item.includes(v.toLowerCase()); }) : this.state.fullSecretList;
```

Comment:
For truly case-insensitive search you need to lowercase both search text and search predicate. In this case it should be:
`let filtered = v ? _.filter(this.state.fullSecretList, (item) => { return item.toLowerCase().includes(v.toLowerCase()); }) : this.state.fullSecretList;`